src/js/scriptlets/subscriber.js

```diff
@@ -49,6 +49,11 @@ if ( typeof vAPI !== 'object' ) {
 /******************************************************************************/
 
 var onMaybeAbpLinkClicked = function(ev) {
+    /* Check if the advanced feature flag is enabled */
+    if (window.store && window.store.getState().advancedModeEnabled === false) {
```

Comment:
maybe just `!window.store.getState().advancedModeEnabled`

Reply:
Just updated it